infra/fendermint/Makefile.toml

```diff
@@ -84,11 +85,14 @@ COMETBFT_SUBDIR = "cometbft"
 CMT_CONTAINER_NAME = "${NODE_NAME}-cometbft"
 FM_CONTAINER_NAME = "${NODE_NAME}-fendermint"
 ETHAPI_CONTAINER_NAME = "${NODE_NAME}-ethapi"
+PROMTAIL_CONTAINER_NAME="${NODE_NAME}-promtail"
```

Comment:
And in the `testnet` for example there will be a different promtail for each container started on the local machine. Should there be only one for the testnet?